overlord/state/change.go

```diff
@@ -423,11 +423,30 @@ func (c *Change) Status() Status {
 	panic(fmt.Sprintf("internal error: cannot process change status: %v", statusStats))
 }
 
+// addNotice triggers a change-update notice.
+func (c *Change) addNotice() error {
+	opts := &AddNoticeOptions{
+		Data: map[string]string{
+			ChangeUpdateNoticeKindDataKey: c.Kind(),
```

Comment:
I am trying to think of value it could add for notice listener, I am sure the spec was written with some use-case in mind but at least in snapd we don't have a good case for it.

CC: @pedronis @Meulengracht

Reply:
I think we need to speak with someone that had a use-case in mind, for now maybe not add it and wait for someone to ask for it? It's easier to add things than remove them.

Reply:
It would make sense only if we thought that we should not expose as fixed that the key is the change-id but I don't think we want not to expose that?  I don't see much harm in adding it though either.

Reply:
I think we have to expose that the key is the change-id as part of the documented behaviour. So let's leave it out for now; as @Meulengracht said, it's much easier to add later if we need it than remove it.